guide/english/linux/10-simple-and-useful-linux-commands/index.md

```diff
@@ -53,5 +53,8 @@ more command allows quickly view file and shows details in percentage. You can p
 10. `cp` Command
 Copy file from source to destination preserving same mode.
 
+11. `df -h` Command
```

Comment:
@tattoodgeek Please change to `df` Command and then mention the `-h` human-readable option in the description.